common.py

```diff
@@ -42,26 +71,28 @@ def checkout_validation_webpage_branch(self):
         - self.GIT_PULL_REQUEST: The state of the GIT CI pull request ("true" or "false").
         - self.GIT_COMMIT: The commit ID of the GIT CI build.
         """
+        try:
+            os.chdir(self.ValidationPath)
 
-        os.chdir(self.ValidationPath)
-
-        webpage_path = os.path.join(self.ValidationPath, "Webpage")
+            webpage_path = os.path.join(self.ValidationPath, "Webpage")
+            if not os.path.isdir(webpage_path):
+                subprocess.run(["git", "clone", f"https://{self.VALIDATION_GIT_PATH}", "--single-branch", "--depth", "1", "-b", self.WEBPAGE_BRANCH, self.WEBPAGE_FOLDER])
+                os.chdir("Webpage")
 
-        if not os.path.isdir(webpage_path):
-            os.system(f"git clone https://{self.VALIDATION_GIT_PATH} --single-branch --depth 1 -b {self.WEBPAGE_BRANCH} {self.WEBPAGE_FOLDER}")
-            os.chdir("Webpage")
+                # Add a default user, otherwise git complains
+                subprocess.run(["git", "config", "user.name", "WCSim CI"])
+                subprocess.run(["git", "config", "user.email", "wcsim@wcsim.wcsim"])

```

Comment:
Changed to:

```
# Add a default user, otherwise git complains
subprocess.run(["git", "config", "user.name", self.GIT_USER])
subprocess.run(["git", "config", "user.email", self.GIT_EMAIL])
```